src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/Create/summary.html.twig

```diff
@@ -110,6 +110,8 @@
             'label': 'Order status'|trans({}, 'Admin.Orderscustomers.Feature'),
           }) }}
 
+          {{ form_rest(summaryForm) }}
```

Comment:
Maybe wrap this in a block, so that it can be extended more easily in a twig template Because the layout is quite minimal with this

Reply:
Agreed 👍 

The rest of the code is ✅